graphics/macgui/macwindowborder.cpp

```diff
@@ -0,0 +1,72 @@
+#include "macwindowborder.h"
```

Comment:
This file is missing our standard header.

Also you must provide relative paths to the includes, so this is "graphics/macgui/macwindowborder.h"